packages/dashboard/src/app/views/templateDetails/karma/templateDetails.karma.js

```diff
@@ -163,140 +147,5 @@ describe('CUJ: Creator can browse templates in grid view: See pre-built template
 
       expect(activePreviewPage).toBeTruthy();
     });
-
-    it('should load the next related template when clicking "View Next Template" button', async () => {
```

Comment:
it'd be awesome if you could retool this so that we can keep the test coverage.

Reply:
yea, I wonder if it makes sense to still have url compatibility with the modal. so if you went to 
`#/templates-gallery?id=59` the template gallery page would load with the modal open to the specific template that matches that id.

Probably best to do as an enhancement ticket if we even want to follow through with it just to not keep changing scope with the current request here.

Reply:
Yeah I had initially thought about that, then completely forgot about asking. I think added that as an enhance would be great.

Reply:
love it, will you make a ticket and assign pea pod?

Reply:
#9996 created.